languages/julia/generic/Parse_julia_tree_sitter.ml

```diff
@@ -2071,33 +2077,25 @@ and map_statement (env : env) (x : CST.statement) : stmt list =
           | `Catch_clause_opt_else_clause_opt_fina_clause
               (clause, elsee, finally) ->
               let v4 = [ map_catch_clause env clause ] in
-              let _v2 =
+              let v5 =
                 match elsee with
-                | Some _x ->
-                    (* map_else_clause env x *)
-                    (* TODO: We can't actually accommodate this within the Generic AST.
-                       "else" is something kind of weird where it's something you enter
-                       if you do not enter the "catch", but it's not the same as a
-                        "finally", which always runs in either case.
-                       https://docs.julialang.org/en/v1/manual/control-flow/#else-Clauses
-                    *)
-                    None
+                | Some x -> Some (map_try_else_clause env x)
```

Comment:
great! that means this is useful not only for Ruby but also Julia! cc @brandonspark